_news-and-blog/2023-03-29-reducing-burden-of-child-care-assistance-applications.md

```diff
@@ -0,0 +1,107 @@
+---
+# Page template info (DO NOT EDIT)
+layout: default
+blog_page: true
+
+# Carousel (Edit this)
+carousel_title: "Reducing burden of child care assistance applications"
+carousel_summary: "Nearly one million American families benefit from federally funded child care assistance. But it’s often difficult for families to navigate an application process that can be confusing and unnecessarily hard."
+# partial path to image
+carousel_image: /news-and-blog/2023-03-29-reducing-burden-of-child-care-assistance-applications-img/carousel-reducing-burden-card.jpg
+# accessibility text for image
+carousel_image_alt_text: ""
```

Comment:
add alt text: Photograph of two women and two young children playing at a table covered in wooden blocks. Text overlaid on the image reads &quot;Reducing child care assistance burdens for families.&quot;"

Reply:
Fixed!